libursa/Cargo.toml

```diff
@@ -47,7 +47,7 @@ sha3 = "0.7.3"
 time = "0.1.36"
 env_logger = "0.5.10"
 blake2b_simd = "0.3.1"
-openssl = { version = "=0.10.12", optional = true }
+openssl = { version = "0.10", optional = true }
```

Comment:
Is this sufficient to ensure a good enough version? 0.9 was published back in February 2018. I just want to make sure that 0.10.0 uses a new enough version of OpenSSL.